scripts/run-vim

```diff
@@ -31,21 +31,11 @@ if [ ! -f "$dir/bin/vim" ]; then
   exit 1
 fi
 
-if [ $coverage -eq 1 ]; then
-  covimerage -q run --report-file /tmp/vim-go-test/cov-profile.txt --append \
+${coverage:+covimerage -q run --report-file /tmp/vim-go-test/cov-profile.txt --append} \
     $dir/bin/vim --noplugin -u NONE -N \
       +"set shm+=WAFI rtp=$dir/share/vim/vimgo packpath=$dir/share/vim/vimgo,$vimgodir" \
       +'filetype plugin indent on' \
       +'packloadall!' \
       "$@"
-else
-  $dir/bin/vim --noplugin -u NONE -N \
-    +"set shm+=WAFI rtp=$dir/share/vim/vimgo packpath=$dir/share/vim/vimgo,$vimgodir" \
-    +'filetype plugin indent on' \
-    +'packloadall!' \
-    "$@"
-fi
-
-
 
```

Comment:
The `/home/travis/gopath/src/github.com/fatih/vim-go/scripts/run-vim: 34: /home/travis/gopath/src/github.com/fatih/vim-go/scripts/run-vim: covimerage: not found` here is really weird.

Reply:
Yeah, not sure what's up with that either... It worked fine on my local machine.

Reply:
Maybe you have bash as `/bin/sh`, while Travis has not - or the other way around?

Reply:
Yeah, it's a link to `bash`; guess this invokes from bash-specific behaviour? Pff...

I think it's okay like this, for now. I have vague plans to rewrite this all to Go one day anyway. Shell scripting is nice to get started, but it very quickly becomes very unwieldy IMHO.

Reply:
Yeah..
I've noticed that your scripts are even affected by having CDPATH defined etc..
As for testing against different Vims you might want to consider https://github.com/tweekmonster/vim-testbed - I am using this with Neomake, and it is nice with CircleCI where jobs can be run in a specific Docker image right away.

I've just released covimerage 0.0.3 with the `-l` option.